src/test/java/test/ConfluenceS3EnabledTest.java

```diff
@@ -0,0 +1,37 @@
+package test;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import test.helm.Helm;
+import test.model.Product;
+
+import java.util.Map;
+
+import static test.jackson.JsonNodeAssert.assertThat;
+
+class ConfluenceS3EnabledTest {
+    private Helm helm;
+
+    @BeforeEach
+    void initHelm(TestInfo testInfo) {
+        helm = new Helm(testInfo);
+    }
+
+
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = {"confluence"})
+    void confluence_s3_storage_env_vars(Product product) throws Exception {
```

Comment:
We can also test a case when only one of the values is defined => and the sys props are not included.

Reply:
Test case added